src/components/CardFilm.js

```diff
@@ -80,12 +87,19 @@ const StyledCard = styled(Card)`
 class Item extends Component {
     constructor(props) {
         super(props);
+
         this.state = {
-            dotsIsVisible: "no-show"
+            dotsIsVisible: "no-show",
+            showModalWindow: false,
         };
 
         this.showDots = this.showDots.bind(this);
         this.hideDots = this.hideDots.bind(this);
+
+        this.showModal = this.showModal.bind(this);
+        this.hideModal = this.hideModal.bind(this);
+
+        this.handleClickTag = this.handleClickTag.bind(this);
```

Comment:
That is not a mistake, but why don't you use [@babel/plugin-proposal-class-properties](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties) plugin ?
It would rather simplify your code if you write your callbacks as class properties.
Frankly, developers have forgotten the `binding` way long ago 🙂

Reply:
changed